app/glossary.md

```diff
@@ -101,6 +113,9 @@ Lead school
 Marketing name
 : The commonly used name of an educational organisation. This may not be the legal name.
 
+Modern languages
+: This term is sometimes used to differentiate the teaching of languages in everyday use from ‘ancient languages’ such as Latin and Ancient Greek which are no longer widely spoken. It also typically does not include teaching English. Avoid using in non-internal communication.
```

Comment:
don't think we can advise avoiding using 'modern languages' as it's all over course name. we currently also use it on Find search

Reply:
It should be getting dropped from the course search soon... 😄. Longer term I’d like to review how it's used in course titles too, as it can mean a few different things...

In the meantime maybe we should downgrade the guidance to something like "use the specific languages where possible" or something?

Reply:
I've updated to "Prefer listing the specific languages where possible".